grimoiressg/__main__.py

```diff
@@ -0,0 +1,25 @@
+from grimoiressg.arguments import parse_arguments_to_initial_context
+from grimoiressg.config import read_config
+from grimoiressg.content_files import recursively_read_files
+from grimoiressg.modules import available_modules
+
+
+def apply_modules(data, config, context):
+    for module in config.get("enabled_modules", []):
+        print(f"Applying module {module}...")
```

Comment:
Is it perhaps useful to use the python logging module?

Reply:
Might be worth it. I'll talk a look.